src/platform/magento2/order/orderProcessor.js

```diff
@@ -0,0 +1,302 @@
+import {productsEquals} from 'vsf-utilities';
+import order from '../../../api/order';
+const orderUtilities = require('./order-utilities');
+const apiUtilities = require('./api-utilities');
+const config = require('config');
+const redis = require('../../../lib/redis');
+const redisClient = redis.getClient(config)
+const logger = console;
+const countryMapper = require('../../../lib/countrymapper')
+
+const TOTAL_STEPS = 4;
+var currentStep = 1;
+
+/**
+ * @param {json} orderData order data in format as described in '../models/order.md'
+ * @returns {Promise}
+ */
+async function cartIdPrepare (orderData) {
+  const api = apiUtilities.getApi(orderData.store_code);
+  const isThisAuthOrder = orderUtilities.isThisAuthOrder(orderData);
```

Comment:
I am not sure if it is really needed - as `orderUtilities.isThisAuthOrder` name still says us so much.